.cspell.json

```diff
@@ -1226,6 +1229,7 @@
     "ptpd",
     "ptree",
     "ptrs",
+    "publically",
```

Comment:
Maybe can be replaced by `publicly`?
https://en.wiktionary.org/wiki/publically

Reply:
`publicly` doesn't occur cspell error.
So, I'll delete the word from `.cspell.json`
```suggestion
```